guava/src/com/google/common/base/Verify.java

```diff
@@ -473,7 +475,8 @@ public static void verify(
    * @see Preconditions#checkNotNull Preconditions.checkNotNull()
    */
   @CanIgnoreReturnValue
-  public static <T> T verifyNotNull(@Nullable T reference) {
+  @EnsuresNonNull("#1")
+  public static <T extends @Nullable Object> @NonNull T verifyNotNull(@Nullable T reference) {
```

Comment:
verifyNotNull should only be called on a non-null reference.  You can just remove all three annotations from this line.